aruco_ros/package.xml

```diff
@@ -20,13 +20,14 @@
   <depend>geometry_msgs</depend>
   <depend>image_transport</depend>
   <depend>rclcpp</depend>
+  <depend>rclcpp_component</depend>
```

Comment:
`rclcpp_components` dependency and alphabetic sort

Reply:
Sounds good

Reply:
I see `aruco` and `aruco_ros` are left to the end, is this intentional?